File: frontend/src/app/shared/service/admin/commun/KeepNoteAdmin.service.ts
import { Injectable } from '@angular/core';
import {HttpClient} from '@angular/common/http';
import {BehaviorSubject, Observable} from 'rxjs';

import {environment} from 'src/environments/environment';

import {KeepNoteDto} from 'src/app/shared/model/commun/KeepNote.model';
import {KeepNoteCriteria} from 'src/app/shared/criteria/commun/KeepNoteCriteria.model';
import {AbstractService} from 'src/app/zynerator/service/AbstractService';


@Injectable({
  providedIn: 'root'
})
export class KeepNoteAdminService extends AbstractService<KeepNoteDto, KeepNoteCriteria> {
     constructor(private http: HttpClient) {
        super();
        this.setHttp(http);
    }

    public constrcutDto(): KeepNoteDto {
        return new KeepNoteDto();
    }

    public constrcutCriteria(): KeepNoteCriteria {
        return new KeepNoteCriteria();
    }

    get API() {
        return environment.apiUrlEmaillingservice + 'admin/keepNote/';
    }
}

File: backend-ms1/backend-ms1/src/main/java/ma/zs/emailling/service/facade/admin/commun/KeepNoteAdminService.java
package ma.zs.emailling.service.facade.admin.commun;

import java.util.List;
import ma.zs.emailling.bean.core.commun.KeepNote;
import ma.zs.emailling.dao.criteria.core.commun.KeepNoteCriteria;
import ma.zs.emailling.zynerator.service.IService;



public interface KeepNoteAdminService extends  IService<KeepNote,KeepNoteCriteria>  {

    List<KeepNote> findByPersonneId(Long id);
    int deleteByPersonneId(Long id);
    long countByPersonneId(Long id);




}
